src/routers/usersRouter.ts

```diff
@@ -0,0 +1,23 @@
+import Koa from "koa"
+import Router from "koa-router"
+import { users } from "../data-access/data-access"
+import { deleteUser } from "../services/crudServices"
+
+export let getUsersRouter = () => {
+    const router: Router = new Router();
+
+    router.get('/getAllUsers');
+
+    router.post('/createUser');
+
+    router.put('/putUser');
+
+    router.get('/user/:id');
+
+    router.delete('/user/:id', (ctx: Koa.Context) => {
+        let deleted = deleteUser(users, ctx.params.id);

```

Comment:
non c'è bisogno sia let visto che non lo sovrascrivi.
Vediti le differenze tra let e const in javascript

Reply:
ah sisi, non ci avevo pensato